src/pages/create/OfferForm.tsx

```diff
@@ -234,8 +264,14 @@ const OfferForm = () => {
           stepProps: {
             offerId,
           },
+          defaultValue: scope,
+        },
+        {
+          ...typeAndThemeStepConfiguration,
+          stepProps: {
+            scope,
+          },
```

Comment:
Yes we do, since we need to access it in the component as well.
We pass scope to StepsForm so it can be used in the `shouldShow`  & `title`.
I tried to delete it and it didn't work anymore ;)

Reply:
https://github.com/cultuurnet/udb3-frontend/pull/496/commits/ee147cb1db52a7353e59dd358ca892b53f2c07ed